README.md

```diff
@@ -11,57 +11,57 @@ password: password
 
 ## Prerequisites
 
-To run the demo locally you have to have installed:
+Install Docker if you don't have it: https://docs.docker.com/desktop/#download-and-install
 
-* mongodb
-* postgresql
-
-or run `docker-compose up`
-
-Also in ENV you have to have following entries:
+Run:
+```bash
+$ docker-compose up -d
+```
+to setup databases.
 
-* MONGO_URL
-* PORT - default to 3000
-* POSTGRES_USER
-* POSTGRES_PASSWORD
-* POSTGRES_PORT - default to 5432
-* POSTGRES_DATABASE - default to 'adminjs'
-* POSTGRES_HOST - default to 'localhost'
-* TYPEORM_USERNAME
-* TYPEORM_PASSWORD
-* TYPEORM_PORT - default to 5432
-* TYPEORM_DATABASE - default to 'adminjs'
-* TYPEORM_HOST - default to 'localhost'
+Make sure your `.env` file is configured. If you didn't do any changes to `docker-compose.yml` file,
+the default contents of the `.env` file should work for you.
 
 ## Starting the app
 
 First, install all dependencies
 
-```
-yarn install
+```bash
+yarn install --frozen-lockfile
 ```
 
 Make sure you have all environmental variables set up (read the previous paragraph).
 
 Then create postgres database and run migrations:
 
+```bash
+$ npx prisma generate     # # this sets up Prisma Client in your node_modules
+$ yarn migration:up
 ```
-yarn sequelize db:migrate
-yarn typeorm migration:run
-yarn mikro-orm migration:up
-npx prisma migrate dev --schema prisma/schema.prisma
+
+Note: If you see the error below when Prisma MySQL migration is run:
+```
+Error: P1017: Server has closed the connection.
 ```
+Please wait a minute or two for the MySQL server to start and retry.
 
 In the end, you can launch the app
 
-```
-yarn start:dev
+```bash
+$ yarn build:watch      # keep it running if developing
```

Comment:
I changed the development steps to require you to build to JS because we will be hosting this app on a remote server soon and if you run the server using TSNode, it will allow you to keep non JS/TS files in your source codebase and run correctly, but if you build using `tsc`, those non JS/TS files won't be moved.
Requiring to build before running ensures that in both dev/prod mode the resulting build is the same.